teste/page-rendez-vous.test.ts

```diff
@@ -0,0 +1,86 @@
+import { expect, test } from '@playwright/test'
+import configurePlaywrightCoverage from 'test-utils/configure-playwright-coverage'
+import { mockRoutesNotion } from './mock-routes/mock-routes-notion'
+
+configurePlaywrightCoverage(test)
+
+test.beforeEach(async ({ page }) => {
+  await mockRoutesNotion(page, {
+    email: 'test@valid.fr',
+    from: 'test',
+    structure: 'entreprise',
+    type: 'contact',
+    needs: 'my needs',
+  })
+})
+
+test('Send rendez-vous demands to api', async ({ page }) => {
+  await test.step('Load page', async () => {
+    await page.goto('/rendez-vous?fromLabel=test')
+  })
+
+  await test.step('Submit without filled data does not work', async () => {
+    await expect(page.getByTestId('rendez-vous-form')).toBeVisible()
```

Comment:
j'aurai opposé le toBeVisible à "not.toBeVisible". 

Ou alors opposer le .toHaveCount(0) à .toHaveCount(1)

Reply:
Pas vraiment possible

tu peux pas avoir not.toBeVisible vu que l'element n'est pas present dans le dom
et avoir toHaveCount(1) est different (peut etre présente dans le dom mais en display none, implementation differente qui ferait que le test passe mais la fonctionnalité est ko)